src/components/ProfilePage/DateJoined.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function DateJoined({ date }) {
+  if (date == null) return null;
+
+  const year = (new Date(date)).getFullYear();
+
+  return (
+    <p className="mb-0">Member since {year}</p>
```

Comment:
Moving forward, can we include i18n with all new code?

Reply:
Will do!